File: src/main/java/com/sstep/recipe/manager/vo/RecipeStepVo.java
package com.sstep.recipe.manager.vo;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;


public class RecipeStepVo {

    private String text;
    private Integer order;

    @JsonCreator
    public RecipeStepVo(
            @JsonProperty("text") final String text,
            @JsonProperty("order") final Integer order) {
        this.text = text;
        this.order = order;
    }

    public String getText() {
        return text;
    }

    public Integer getOrder() {
        return order;
    }

    @Override
    public String toString() {
        return "RecipeStepVo{" +
                "text='" + text + '\'' +
                ", order=" + order +
                '}';
    }
}

File: src/main/kotlin/com/sstep/recipe/manager/vo/RecipeStepVo.kt
package com.sstep.recipe.manager.vo

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
data class RecipeStepVo @JsonCreator constructor(
    val order: Int? = null,
    val text: String? = null,
    val imageId: String? = null
)